include/zephyr/bluetooth/bluetooth.h

```diff
@@ -1424,24 +1424,42 @@ int bt_le_ext_adv_create(const struct bt_le_adv_param *param,
 
 struct bt_le_ext_adv_start_param {
 	/**
-	 * @brief Advertiser timeout (N * 10 ms).
+	 * @brief Maximum advertising set duration (N * 10 ms)
+	 *
+	 * The advertising set can be automatically disabled after a certain
+	 * amount of time has passed since it first appeared on air.
 	 *
-	 * Application will be notified by the advertiser sent callback.
-	 * Set to zero for no timeout.
+	 * Set to zero for no limit. Set in units of 10ms.
+	 *
+	 * When the advertising set is automatically disabled because of this
+	 * limit, @ref bt_le_ext_adv_cb.sent will fire.
 	 *
 	 * When using high duty cycle directed connectable advertising then
 	 * this parameters must be set to a non-zero value less than or equal
 	 * to the maximum of @ref BT_GAP_ADV_HIGH_DUTY_CYCLE_MAX_TIMEOUT.
 	 *
 	 * If privacy @kconfig{CONFIG_BT_PRIVACY} is enabled then the timeout
 	 * must be less than @kconfig{CONFIG_BT_RPA_TIMEOUT}.
+	 *
+	 * See Core 4.E.7.8.56 parameter "Duration" for background information.
```

Comment:
Please spell out specific version and other details. This helps in locating the correct thing.

```suggestion
	 * See parameter "Duration" in Core Specification Version 5.4 Vol. 4 Part E, Section 7.8.56, for background information.
```

Reply:
Agree (and please use version 6.0 instead of 5.4)

Reply:
In this file we refer to 5.4, so I suggested 5.4, generally core spec section numbers do not change across revisions.

Reply:
Hmm, I always though we would refer to the newest version of the spec available when adding the reference. We don't have any guidelines on this though

Reply:
Ok. It does not matter that much because section numbers don't change in core specifications across versions. Just FYI this may not be true for other profiles and spec (certainly not true for Mesh).

Reply:
@Thalley will it be worth creatin an issue regarding this and using that issue as basis, update coding guidlines to specifically state this?

Reply:
> Just FYI this may not be true for other profiles and spec (certainly not true for Mesh).

Yeah, I've noticed, which is also why it's important to provide the version number :) Arguably we could omit it for the BT Core spec, as it doesn't change, but there still needs to be a minimum version where it was added